User Guides/10-Configure Power BI.md

```diff
@@ -4,21 +4,24 @@
 
-## Install the PowerBI on-premise Data Gateway
+## Log into the SQL Server VM
 
-> TIP: Before downloading anything, you will want to disable the Enhanced Network Security on this VM. To do this, click **Server Manager** > **Local Server** and turn **IE Enhanced Security Configuration** to **Off** for both "Administrators" and "Users". Click **OK** to apply the changes and close **Server Manager**. This isn't required, but otherwise you will have to allow every individual download.
+Now, log into the SQL Server VM in your solution. Technically, the PowerBI Data Gateway could be installed on any of the VMs deployed as part of your solution, or even a new one, but the SQL Server VM is a good choice since it is not heavily used. You can find this VM by opening the [Azure portal](https://portal.azure.com/) and clicking **Resource groups**. Type in the name you gave for your solution, which is also used as the name of the resource group, to find your resource group. Click it, and sort the resources by type. Scroll down to the virtual machines and find the one ending in "sqlvm00". Click on this VM, then click the **Networking** link. The private IP listed is the IP address you will use to connect to the SQL Server VM.
 
-Using Internet Explorer on the VM (which comes pre-installed) browse to https://powerbi.microsoft.com/en-us/gateway, click **DOWNLOAD GATEWAY**, and click **Run** to begin the download. Click **Next** for all default options, accept the terms of agreement, and click **Install**. The installation takes a few minutes. Next, you need to register your gateway by entering an email address and clicking **Sign in**.
+> NOTE: Because your VM is running in a VNET, you must first ensure that your VPN client is running before you can connect to your VM. Until it is running, you will not be able to connect.
 
-Now that you are signed in, click **Next** and give your on-premises data gateway a name and recovery key and click **Configure**. Your gateway is now ready.
+Log in to the SQL Server VM using your preferred client, such as Remote Desktop, with the private IP address and the username and password you used for your deployment. If you are using Remote Desktop, select **More choices** > **Use a different account** to enter the username and password.
+
+## Upload the PowerBI Data Gateway installer and install
 
+Drag the installer executable from your local environment onto the desktop of the VM. Once there, double click it to begin the installation. Click **Next** for all default options, accept the terms of agreement, and click **Install**. The installation takes a few minutes. Next, you need to register your gateway by entering an email address and clicking **Sign in**.

```

Comment:
I don't think Remote Desktop supports drag-and-drop for local files (or at least older versions do not). The easiest way to do the upload is using copy/paste semantics, but some setup may be required in the Remote Desktop client

Reply:
My two cents: When connecting to using RDC, I always map a drive using RDC Dialog -> Local Resources -> "More...". Drag and drop is not supported AFAIK.

Reply:
Good catch. I've instructed to copy/paste the installer executable, rather than drag/drop.